devpack/src/main/java/io/neow3j/devpack/Storage.java

```diff
@@ -46,6 +46,56 @@
     @Instruction(interopService = SYSTEM_STORAGE_GET)
     public static native ByteString get(StorageContext context, byte[] key);
 
+    /**
+     * Returns the value corresponding to the given key and converts it to a byte array.
+     * <p>
+     * This incurs the GAS cost of converting the {@code ByteString} value to a byte array.
+     *
+     * @param context The storage context to search in.
+     * @param key     The key to search for.
+     * @return the value corresponding to the given key converted to a byte array.
+     */
+    @Instruction(opcode = OpCode.SWAP)
+    @Instruction(interopService = SYSTEM_STORAGE_GET)
+    @Instruction(opcode = OpCode.CONVERT, operand = StackItemType.BUFFER_CODE)
+    public static native byte[] getByteArray(StorageContext context, byte[] key);
+
+    /**
+     * Returns the value corresponding to the given key as a string.
+     * <p>
+     * This does not incur any extra GAS costs.
+     *
+     * @param context The storage context to search in.
+     * @param key     The key to search for.
+     * @return the value corresponding to the given key as a string.
+     */
+    @Instruction(opcode = OpCode.SWAP)
+    @Instruction(interopService = SYSTEM_STORAGE_GET)
+    @Instruction()
```

Comment:
It should work without this empty `Instruction`. Did you try?

Ah and then you can actually simply have only one `Instruction` with the interopService attribute set. The SWAP is taken care of the compiler itself.

Same for the other String related methods

Reply:
Good input, yes, it works!